app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
```

Comment:
- [ ] `new` action is missing in your comments controller. Kindly note that this activity requires that you explicitly declare both `new` and `create` actions for comments controller. Kindly ensure to improve upon the same, which should be similar to:
```ruby
# controllers/comments_controller.rb

class CommentsController < ApplicationController
  ...

  def new
    @comment = Comment.new
  end

  def create
  
    ...
  end


  private 
  def comment_params
    params.require(:comment).permit(:text)
  end

	...
end